src/components/Header.js

```diff
@@ -0,0 +1,22 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] Please do not disable eslint checks. Eslint helps improve your Javascript's quality and keeps it to standard.